src/components/sectionList/SectionListWrapper.tsx

```diff
@@ -76,33 +79,53 @@ export const SectionListWrapper = <Model extends IdentifiableObject>({
         <div>
             <SectionListTitle />
             <FilterWrapper>{filterElement}</FilterWrapper>
-            <SelectionListHeader />
-            <SectionList
-                headerColumns={headerColumns}
-                onSelectAll={handleSelectAll}
-                allSelected={allSelected}
-            >
-                <SectionListMessage />
-                {data?.result.map((model) => (
-                    <SectionListRow
-                        key={model.id}
-                        modelData={model}
-                        selectedColumns={headerColumns}
-                        onSelect={handleSelect}
-                        selected={selectedModels.has(model.id)}
-                        renderColumnValue={({ path }) => {
-                            return (
-                                <ModelValue
-                                    path={path}
-                                    schema={schema}
-                                    sectionModel={model}
-                                />
-                            )
-                        }}
-                    />
-                ))}
-                <SectionListPagination data={data} />
-            </SectionList>
+            <div className={css.listDetailsWrapper}>
+                <div className={css.listToolbarWrapper}>
```

Comment:
Just a nitpick: This div isn't necessary, you could assign it to its own area. I don't care too much

Reply:
Not sure which one you are referring to? Pretty sure both are required for the required functionality. If listToolbarWrapper was removed the toolbar-actions would be affected by the listDetails-grid (that is supposed to target the list and details). These also handle the responsive functionality of the list.

Reply:
See here: #364

Reply:
Thanks, that makes sense! Didn't understand the "assign it to it's own area" as adding it to the grid. Will try it out!